app/src/main/res/layout/activity_sign_in.xml

```diff
@@ -125,25 +125,12 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent" />
 
-        <!--TextView
-            android:id="@+id/connect_with_facebook_sign_in"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="415dp"
-            android:letterSpacing="0.2"
-            android:text="@string/uc_connect_with_facebook"
-            android:textStyle="bold"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent" /-->
-
         <com.facebook.login.widget.LoginButton
             android:id="@+id/login_button"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_gravity="center_horizontal"
             android:layout_marginTop="415dp"
```

Comment:
refactor this, and use @dimen/

Reply:
Thanks for your suggestion. I am going to improve it